app/src/main/res/values/strings.xml

```diff
@@ -468,32 +468,50 @@
 
     <!-- Onboarding: Title -->
     <!-- %1$s will be replaced by the app name and build variant(e.g. Firefox Focus, Firefox Klar Nightly) -->
-    <string name="onboarding_title">Welcome to %1$s!</string>
+    <string name="onboarding_title" moz:removedIn="106">Welcome to %1$s!</string>
+
+    <string name="onboarding_close_button_content_description">Close</string>
+    <string name="onboarding_first_screen_title">Welcome to %1$s</string>
+    <string name="onboarding_first_screen_subtitle">Fast. Private. No distractions.</string>
+    <string name="onboarding_first_screen_button_text">Get started</string>
```

Comment:
All new strings need a description to help localizers also understand the context of where each string is used.

Reply:
done